pkg/logs/tailers/socket/tailer.go

```diff
@@ -67,7 +68,10 @@ func (t *Tailer) forwardMessages() {
 	}()
 	for output := range t.decoder.OutputChan {
 		if len(output.GetContent()) > 0 {
-			t.outputChan <- message.NewMessageWithSource(output.GetContent(), message.StatusInfo, t.source, output.IngestionTimestamp)
+			origin := message.NewOrigin(t.source)
+			source_host_tag := fmt.Sprintf("source_host:%s", t.Conn.RemoteAddr().String())
+			origin.SetTags(append(t.source.Config.Tags, source_host_tag))
```

Comment:
`append` can reuse slice if it has capacity, so multiple calls to append on the same slice can overwrite each other. Is it safe to call on `t.source.Config.Tags` here?